### ISSUE TITLE
Add feature plots to library

### ISSUE THREAD

**Issue by bnaul** [mltsp/mltsp#131]
Right now, the plots are generated on the front end; as we move towards a new plotting framework, it would be nice to have pure Python functions that generate feature summary plots from the backend. They'd make for nice additions to example notebooks.

**Comment by stefanv** [comment]
Either plot.ly or bokeh should give us that functionality essentially for
free.

**Comment by bnaul** [comment]
Yep! Mostly just a reminder to incorporate it into the backend tutorials
whenever we make the switch.

**Comment by Timereversal** [comment]
is there any tutorial related to bokeh and cesium interaction ?   i would like to click on Cesium entity then generate a graph using bokeh.
